### PR TITLE
    - fix problems with tags with multi-line attributes

### LINKED ISSUE #562: suggestions for a tag with attributes on multiple lines wrong sometimes
When a tags attributes are spread across multiple lines in a self closing tag, and a suggestions is triggered between 2 spaces, the suggestion result is always for the element tag.

### LINKED ISSUE #563: The hiddenGroupRef is erroneously supplying a carriage return.

### PR DESCRIPTION
**Pull request by rthomas320**
- remove an incorrect newline in the hiddenGroupRef snippet

    Closes #562
    Closes #563

### CODE REVIEW

File: src/language/providers/utils.ts [hunk 1]
```diff
@@ -276,14 +276,15 @@ export function checkMultiLineTag(
     while (currentText.trim() === '' || !currentText.includes('<')) {
       --currentLine
       currentText = document.lineAt(currentLine).text
-      if (!currentText.endsWith('/>')) {
-        break
-      }
+      //      if (!currentText.endsWith('/>')) {
+      //        break
+      //      }
```
Comment (scholarsmate) [inline src/language/providers/utils.ts:281]: Remove commented code.

**Review by scholarsmate — APPROVED** [review]
+1, the code here looks sane to me.

File: src/language/providers/utils.ts (hunk 1 above)
Comment (rthomas320) [reply src/language/providers/utils.ts]: I wasn't sure if I needed, I was going to remove after testing, but forgot. It's removed now.

**Review by Shanedell — APPROVED** [review]
+1 👍

**Pull request merged**